examples/all_features/stringification.cpp

```diff
@@ -75,6 +75,33 @@ struct Foo
     friend bool operator==(const Foo&, const Foo&) { return false; }
 };
 
+struct MyStringType
+{
+    std::string s;
+    template <size_t N> MyStringType(const char (&in)[N]) : s(in, in+N) {}
+    friend bool operator==(const MyStringType &lhs, const MyStringType &rhs) { return lhs.s == rhs.s; }
+};
+
+template <typename T, typename K>
+std::ostream& operator<<(std::ostream& stream, const MyType<T, K>& in) {
+    stream << "[" << in.one << ", " << in.two << "]";
+    return stream;
+}
```

Comment:
this doesn't seem related to `MyStringType`

Reply:
indeed, it was moved inadvertently and has nothing to do with `MyStringType`.